include/stdexec/__detail/__meta.hpp

```diff
@@ -490,6 +490,12 @@ namespace stdexec {
   template <class _Ty>
     using __id = __minvoke<__id_<__has_id<_Ty>>, _Ty>;
 
+  template <class _Ty>
+    using __cvref_t = __copy_cvref_t<_Ty, __t<std::remove_cvref_t<_Ty>>>;
+
+  template <class _From, class _To>
+    using __cvref_id = __copy_cvref_t<_From, __id<std::remove_cvref_t<_To>>>;
```

Comment:
I don't think the `remove_cvref_t` is necessary. I think it's a bug if `_To` is ever cvref-qualified here. Maybe see if that's true and if so, drop the `remove_cvref_t`.

Reply:
The static_assert I added should detect if `_To` is cvref-qualified. Indeed, the `remove_cvref_t` is not needed.